app/dashboard/src/pages/authentication/Login.tsx

```diff
@@ -100,6 +100,7 @@ export default function Login() {
                 size="large"
                 variant="outline"
                 icon={GoogleIcon}
+                disableMask
```

Comment:
`Button` component allows to pass as icon a custom component. Can we use that instead of introducing extra props?

Reply:
oh right... good catch